### ISSUE TITLE
A custom shader workflow like Unity, Unreal or other engines

### ISSUE THREAD

**Issue by wangtieyi** [armory3d/armory#2012]
You can write shader with glsl and .json, then they become custom node in Blender and can be applied to multiple projects with different textures

**Comment by onelsonic** [comment]
Yep, I just started to look into GLSL and this will be like the holy grail for shaders.
I have my eye on this but it is not out in the market yet...
http://www.shaderplay.com/products/nodeflex/overview/overview.html
